Dockerfile.build

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.12-alpine3.9
```

Comment:
Let's keep `build/` as is please. It's mean for performance of `docker build`. It prevents having to pass copy the whole tree, only what is in `build/` is copied to build context, and also ensure cache works is used as long as nothing in `build/` changed.

Reply:
The bottleneck is not passing the whole tree, that takes seconds. Let's have a call about this.

Reply:
To clarify for anyone reading this:
- copying context tool long time because of `vendor/`
- modules provide dependency caching, so no need to use `vendor/`